serialize_property_test.ts

```diff
@@ -348,7 +349,8 @@ test({
     class Test2 extends Serializable {
       @SerializeProperty({
         serializedKey: "serialize_me_2",
-        fromJsonStrategy: (json) => new Test1().fromJson(json),
+        fromJsonStrategy: (json: JsonValue) =>
```

Comment:
Typing here is implied by the `SerializePropertyArgument` -> `FromJsonStrategy` types, no need to manually specify it
```suggestion
        fromJsonStrategy: (json) =>
```

![Screen Shot 2020-11-03 at 20 00 13](https://user-images.githubusercontent.com/49878815/98057066-5edfcc80-1e0f-11eb-804e-a9483c42b8fc.png)